applications/tari_merge_mining_proxy/src/main.rs

```diff
@@ -43,6 +43,143 @@ use std::convert::Infallible;
 use structopt::StructOpt;
 use tari_common::{configuration::bootstrap::ApplicationType, ConfigBootstrap, GlobalConfig};
 
+#[cfg(feature = "winservice")]
```

Comment:
This can be cleaner if it's split into 2 files and a single `#cfg`

Reply:
Have used `[[bin]]` in cargo.toml to build separate executables, removed most of this clutter.